apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -452,7 +530,47 @@ type ApplyStrategy struct {
 	// +kubebuilder:validation:Optional
 	ServerSideApplyConfig *ServerSideApplyConfig `json:"serverSideApplyConfig,omitempty"`
 
-	// TakeoverAction describes the action to take when we place the selected resources on the target cluster the first time.
+	// TakeoverAction determines the action to take when Fleet applies resources to a member
+	// cluster for the first time and finds out that the resource already exists in the cluster.
+	//
+	// This setting is most relevant in cases where you would like Fleet to manage pre-existing
+	// resources on a member cluster.
+	//
+	// Available options include:
+	//
+	// * AlwaysApply: with this action, Fleet will apply the hub cluster manifests to the member
+	//   clusters even if the affected resources already exist. This is the default action.
+	//
+	//   Note that this might lead to some fields being overwritten on the member clusters.
+	//
+	// * ApplyIfNoDiff: with this action, Fleet will apply the hub cluster manifests to the member
+	//   clusters if (and only if) pre-existing resources look the same as the hub cluster manifests.
+	//   This is a safer option as pre-existing resources that are inconsistent with the hub cluster
+	//   manifests will not be overwritten; in fact, Fleet will ignore them until the inconsistencies
+	//   are resolved properly: any change you make to the hub cluster manifests would not be
+	//   applied, and if you delete the manifests or even the ClusterResourcePlacement itself
+	//   from the hub cluster, these pre-existing resources would not be taken away.
+	//
+	//   Fleet will check for inconsistencies in accordance with the DiffMode setting. See also
+	//   the comments on the DiffMode field for more information.
+	//
+	//   If a diff has been found in a field that is managed by Fleet (i.e., the field is specified
+	//   in the hub cluster manifest), consider one of the following actions:
+	//   * set the field in the member cluster to be of the same value as that in the hub cluster
+	//     manifest.
+	//   * update the hub cluster manifest so that its field value matches with that in the member
+	//     cluster.
+	//   * switch to the AlwaysApply action, which will allow Fleet to overwrite the field with the
+	//     value in the hub cluster manifest.
+	//
+	//   If a diff has been found in a field that is not managed by Fleet (i.e., the field is not
+	//   specified in the hub cluster manifest), consider one of the following actions:
+	//   * remove the field from the member cluster.
+	//   * update the hub cluster manifest so that the field is included in the hub cluster manifest.
+	//
+	//   If appropriate, you may also delete the object from the member cluster; Fleet will recreate
+	//   it using the hub cluster manifest.
```

Comment:
duplicated comments?

Reply:
Hi Ryan! These are not duplicates; one is for managed fields and the other for unmanaged fields -> I have add emphasis to make things more clear

Reply:
Resolving as the change no longer applies.